src/main/java/gov/loc/repository/bagit/writer/BagitFileWriter.java

```diff
@@ -41,5 +64,13 @@ public static void writeBagitFile(final Version version, final Charset encoding,
     final String secondLine = "Tag-File-Character-Encoding : " + encoding + System.lineSeparator();
     logger.debug("Writing line [{}] to [{}]", secondLine, bagitPath);
     Files.write(bagitPath, secondLine.getBytes(StandardCharsets.UTF_8), StandardOpenOption.WRITE, StandardOpenOption.APPEND);
+    
+    if(version.compareTo(ONE_DOT_ZERO) >= 0 && payloadByteCount != null && payloadFileCount != null){ //if it is 1.0 or greater
+      final String thirdLine = "Payload-Byte-Count : " + payloadByteCount + System.lineSeparator();
```

Comment:
I'm wondering whether now is the right time to fix the naming convention here - `firstLine`, et al. are making me itch a bit. 

Also, looking at the `Files.write` code, this is repeatedly opening the same file with a bunch of redundant options and the getBytes calls are being repeated on every line as well. Could all of this be collapsed into a single array with 2-4 lines for a single [`Files.write`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#write-java.nio.file.Path-java.lang.Iterable-java.nio.charset.Charset-java.nio.file.OpenOption...-) call using the form which accepts text and the target output encoding?